### DISCUSSION TITLE
Dark mode for email text

### DISCUSSION THREAD

**Discussion by andreashe** [protz/thunderbird-conversations#1922] [Ideas]
At the moment it displays the text with white BG even if darkmode is on in Windows. Would be great to adapt the dark mode for text-wrting-area as well.

**Comment by johnnymast** [comment]
I would love this as well.

**Comment by Standard8** [comment]
Dark mode is https://github.com/thunderbird-conversations/thunderbird-conversations/projects/5

As I've said in a few places, I'm happy for patches, but I don't really have time to work on this. There are still regressions from the Thunderbird 115 upgrade that I haven't had time to get to.

**Comment by SylwesterZarebski** [comment]
It looks like it is implemented: https://github.com/thunderbird-conversations/thunderbird-conversations/issues/1611

**Comment by wKovacs** [comment]
I discoverd the button to toggle dark mode for individuel messages. Interesstingly I have to DISable dark mode to make the background dark.

Could we get a global option were we can set the default for all mails? This way we only need to toggle "buggy" mails instead of every single time we open a mail
